src/matrix_viz/core.clj

```diff
@@ -193,8 +194,8 @@
      :legend-padding      legend-padding
      :legend-top          (+ image-height-no-legend legend-padding)
      :legend-width        (- image-width (* 2 legend-padding))
-     :legend-min          (reduce min (matrix-values nodata-value matrix))
-     :legend-max          (reduce max (matrix-values nodata-value matrix))}))
+     :legend-min          (reduce min v0 (matrix-values nodata-value matrix))
+     :legend-max          (reduce max v0 (matrix-values nodata-value matrix))}))
```

Comment:
@lambdatronic we hadn't caught this bug when last reviewing this PR - it's now fixed.